bin/mev/src/cmd/build/reth_ext.rs

```diff
@@ -0,0 +1,139 @@
+        let (service, builder) =
+            Service::from(build_config, None, pool.clone(), provider.clone(), chain_spec.clone())
+                .unwrap();
+
+        if self.emulate_fcu {
+            let builder = builder.clone();
+            let fut = Box::pin(async move {
+                let clock = clock::for_sepolia();
+                loop {
+                    let start = SystemTime::now();
+                    let timestamp =
+                        start.duration_since(UNIX_EPOCH).expect("Time went backwards").as_secs();
+                    let block_number = provider.best_block_number().unwrap();
+                    let parent = provider.block_by_number(block_number).unwrap().unwrap();
+                    let parent = parent.hash_slow();
+                    let attributes = PayloadAttributes {
+                        timestamp: timestamp.into(),
+                        prev_randao: H256::default(),
+                        suggested_fee_recipient: Address::default(),
+                        withdrawals: Some(vec![]),
+                        parent_beacon_block_root: None,
+                    };
+                    let payload_attributes = PayloadBuilderAttributes::new(parent, attributes);
+                    let job =
+                        builder.new_payload_job(payload_attributes).expect("no errors for now...");
+                    // NOTE: currently, the way we are building `mempool_builder::Job`,
+                    // we just need to drive the future here and then do nothing else.
+                    job.await.expect("no errors");
+
+                    let duration = clock.duration_until_next_slot();
+                    tokio::time::sleep(duration).await;
+                }
+            });
+            executor.spawn(fut);
+        }
+
+        let (payload_service, payload_builder) = PayloadBuilderService::new(builder);
+
+        let fut = Box::pin(async move {
+            match service.spawn().await {
+                Ok(handle) => match handle.await {
+                    Ok(()) => (),
+                    Err(err) => {
+                        tracing::error!(err = ?err);
+                    }
+                },
+                Err(err) => {
+                    tracing::error!(err = ?err, "could not launch mempool builder");
+                }
+            }
+        });
+
+        executor.spawn_critical("mempool builder", fut);
+        executor.spawn_critical("payload builder service", Box::pin(payload_service));
+
+        Ok(payload_builder)
+    }
+}
+
+pub(crate) async fn launch_reth(ext: RethNodeExt) {
+    let task_manager = TaskManager::new(tokio::runtime::Handle::current());
+    let task_executor = task_manager.executor();
+    let ctx = CliContext { task_executor };
+
+    let mut node = NodeCommand::<RethExt>::parse_from([
+        "",
+        "--chain",
+        "sepolia",
+        "--http",
+        "--authrpc.jwtsecret",
+        "/secrets/jwt.hex",
```

Comment:
@ralexstokes TODO can we drop these (ideally they are assumed defaults from EL-CL pair)

Reply:
if not, let's add as a config option